doc/install/python.rst

```diff
@@ -46,14 +41,28 @@ to see information about the version and enabled features:
     >>> print(arbor.__config__)
 
 You are now ready to use Arbor! You can continue reading these documentation pages, have a look at the
-:ref:`Python API reference<pyoverview>` , or visit the :ref:`Quick Start page<gs_single_cell>`.
+:ref:`Python API reference<pyoverview>`, or visit the :ref:`gs_other_examples`.
 
 .. Note::
     To get help in case of problems installing with pip, run pip with the ``--verbose`` flag, and attach the output
     (along with the pip command itself) to a ticket on `Arbor's issues page <https://github.com/arbor-sim/arbor/issues>`_.
 
+Customising Arbor
+^^^^^^^^^^^^^^^^^
+
+If you want to work on Arbor's code, you can get a copy of our repo and point `pip` at the local directory:
+
+.. code-block:: bash
+
+    # get your copy of the Arbor source
+    git clone https://github.com/arbor-sim/arbor.git --recursive
+    # make your changes and then instruct pip to build and install the local source
+    pip3 install ./arbor/
```

Comment:
Have you tested this?

Does it work with pip v20.3 and earlier?

The more that I think about it, the better I think it is to advertise using CMake here
* with pip it does a fresh configure and build from scratch every time, which is very time consuming (no iterative builds)
* unless you pass `--verbose` to pip, you get no feedback about the build.

I would also add a very strong (emphasize using a text box) hint that a venv should be used, otherwise `pip3 install` won't work without sudo, and we don't want to ask people to call `sudo` (too much power)

Reply:
I tested it (naturally :) ). Since new pip installs as user package if it does not find the system package dir writable, I consider this solved/clear. Presumably they are already familiar with pip.

Wether or not a venv is used is also up to the user imho, I don't want to prescribe venv over conda environments are whatever pops up next in the landscape.